modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/SAMLIdPInitiatedSLOTestCase.java

```diff
@@ -304,4 +303,15 @@ protected static boolean assertIfLogExists(LogViewerClient logViewerClient, Stri
         }
         return matchFound;
     }
+
+    private SAMLSSOServiceProviderDTO getServiceProvider(SAMLConfig samlConfig) throws RemoteException, IdentitySAMLSSOConfigServiceIdentityException {
+
+        SAMLSSOServiceProviderDTO[] samlssoServiceProviderDTOs = ssoConfigServiceClient.getServiceProviders().getServiceProviders();
+        for (SAMLSSOServiceProviderDTO spDTO : samlssoServiceProviderDTOs) {
+            if (spDTO.getIssuer().equals(samlConfig.getApp().getArtifact())) {
+                return spDTO;
+            }
+        }
+        return null;
+    }
```

Comment:
Fix character length

Reply:
Fixed - https://github.com/wso2/product-is/pull/7059